examples/regression/4-tutorials/plot_ts-tutorial.py

```diff
@@ -0,0 +1,334 @@
+"""
+========================
+Tutorial for time series
+========================
+
+In this tutorial we describe how to use
+:class:`mapie.time_series_regression.MapieTimeSeriesRegressor`
+to estimate prediction intervals associated with time series forecast.
+
+We use here the Victoria electricity demand dataset used in the book
+"Forecasting: Principles and Practice" by R. J. Hyndman and G. Athanasopoulos.
+The electricity demand features daily and weekly seasonalities and is impacted
+by the temperature, considered here as a exogeneous variable.
+
+Before estimating prediction intervals with MAPIE, we optimize the base model,
+here a Random Forest model. The hyper-parameters are
+optimized with a :class:`sklearn.model_selection.RandomizedSearchCV` using a
+sequential :class:`sklearn.model_selection.TimeSeriesSplit` cross validation,
+in which the training set is prior to the validation set.
+
+Once the base model is optimized, we can use
+:class:`MapieTimeSeriesRegressor` to estimate
+the prediction intervals associated with one-step ahead forecasts through
+the EnbPI method [1].
+
+As its mother class :class:`MapieRegressor`, :class:`MapieTimeSeriesRegressor`
+has two main arguments : "cv", and "method".
+In order to implement EnbPI, "method" must be set to "enbpi" (the default
+value) while "cv" must be set to the :class:`mapie.subsample.BlockBootstrap`
+class that block bootstraps the training set.
+This sampling method is used in [1] instead of the traditional bootstrap
+strategy as it is more suited for ime series data.
+
+The EnbPI method allows you update the residuals during the prediction,
+each time new observations are available o that the deterioration of
+predictions, or the increase of noise level, can be dynamically taken into
+account. It can be done with :class:`MapieTimeSeriesRegressor` through
+the ``partial_fit`` method called at every step.
+
```

Comment:
Do we want to set all the parameters using: "methods" or `methods`?

Reply:
``partial_fit`` is here a class method (not the argument method), I added "class method" fo clarification.